### PR TITLE
Add blog for metadata links

### PR DESCRIPTION
**Pull request by sonalkr132**
[Read with markdown formatting](https://gist.github.com/sonalkr132/8608c421ae1cf79623567a05a3bffaf0#file-and-then-there-was-one-md)

[metadata milestone](https://github.com/rubygems/rubygems.org/milestone/7)

### CODE REVIEW

**Comment by kbrock** [comment]
Looking good.

The reason I like this feature is because when you author a gem, you have full control over the metadata rather than needing to login to a website to set the metadata.

What a wonderful group effort

File: _posts/2019-03-05-and-then-there-was-one-metadata-links.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: And then there was one
```
Comment (dwradcliffe) [inline _posts/2019-03-05-and-then-there-was-one-metadata-links.md]: Let's use a more descriptive title

Comment (sonalkr132) [reply _posts/2019-03-05-and-then-there-was-one-metadata-links.md]: updated to:
> Removing the edit linkset form

Comment (sonalkr132) [reply _posts/2019-03-05-and-then-there-was-one-metadata-links.md]: a more descriptive but rather wordy one could be:
> Removing the edit linkset form and introducing metadata URI attributes.

**Pull request merged**